.github/workflows/create-ubuntu-distribution-package.yml

```diff
@@ -0,0 +1,81 @@
+name: Build & Package (Ubuntu)
+
+on:
+  push:
+    tags:
+      - 'v*'
+  workflow_dispatch:
+  pull_request:
+    branches: [ main, master ]
```

Comment:
I don't think we need the master part. It's obsolete and our repo doesn't have it.